Apps/Comp_Testing_Info.md

```diff
@@ -0,0 +1,99 @@
+# 1.
+The driver requires a dso for the component you are running. To ensure this file is produced, make sure that `TYPE SHARED` is listed as a dependency in the CMakeLists.txt for the component. Now, when the model is built again the file should be produced in the "lib" directory within the install directory within your build directory.
+# 2.
+Add a SetServices subroutine at the bottom of the component file, after the end of the module. For example:
+```
+subroutine SetServices(gc, rc)
+  use ESMF
+  use SS2G_GridCompMod, only : mySetservices=>SetServices
+  type(ESMF_GridComp) :: gc
+  integer, intent(out) :: rc
+
+  call mySetServices(gc, rc=rc)
+end subroutine SetServices
+```
+# 3.
+Set up your configuration file. Note that the captured files follow the following format: `<component>_<state>_<before/after>_runPhase<1/2>`. This should be named COMP.rc and contain the following labels:
+
```

Comment:
I wonder if this should be a separate file and then you just reference it in your instructions.   I say this only because it is rather long and lets the user lose track of the flow of the instructions.   There may not be a good solution to this.

Reply:
I agree, it is a bit long for instructions. I'll have to think of a good way to maintain the flow, which may include adding the config file to that directory and referencing it, as you said.